flexmeasures/ui/templates/crud/account.html

```diff
@@ -146,21 +146,16 @@ <h3>Edit {{ account.name }}</h3>
     </div>
     <div class="col-md-8">
       <div
-        id="toastcontainer"
-        class="toast align-items-center text-bg-primary border-0"
+        class="toast-container"
```

Comment:
Why this `class`? One level below we have an element with almost the same `id` name.

It seems you can remove the class, as it is not used.

Why do we need one level above the toast container anyways? The Bootstrap example seems simpler: 

```
<div class="toast-container">
  <div class="toast" role="alert" aria-live="assertive" aria-atomic="true">
```

And `role="alert"` is only being applied to some Toasts, not wrapping elements or containers.

Let's make this a bit simpler.

Reply:
i think i can make this simpler, I can probably merge the classes